From 536686e4149c79993c00f1a1244e53717a4f2b8e Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Wed, 16 Nov 2016 13:02:04 +0100
Subject: [PATCH 1/2] native: Increase ART heap limit to 192MB for 1024MB RAM
 devices

 * Needed for Google SetupWizard to start

 * Without this, SetupWizard is compiled by ART on boot
   and runs out of memory (heap) while running, resulting
   in the SetupWizard crashing and blocking boot

Change-Id: Ic5b8caf4352f6d104af9121a09a01adad1c5bcc3
Signed-off-by: Adrian DC <radian.dc@gmail.com>
Signed-off-by: sub77 <sub77@ymail.com>
---
 build/phone-xhdpi-1024-dalvik-heap.mk     | 2 +-
 build/tablet-7in-hdpi-1024-dalvik-heap.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/phone-xhdpi-1024-dalvik-heap.mk b/build/phone-xhdpi-1024-dalvik-heap.mk
index 221227d..5707b1d 100644
--- a/build/phone-xhdpi-1024-dalvik-heap.mk
+++ b/build/phone-xhdpi-1024-dalvik-heap.mk
@@ -18,7 +18,7 @@
 
 PRODUCT_PROPERTY_OVERRIDES += \
     dalvik.vm.heapstartsize=8m \
-    dalvik.vm.heapgrowthlimit=96m \
+    dalvik.vm.heapgrowthlimit=192m \
     dalvik.vm.heapsize=256m \
     dalvik.vm.heaptargetutilization=0.75 \
     dalvik.vm.heapminfree=512k \
diff --git a/build/tablet-7in-hdpi-1024-dalvik-heap.mk b/build/tablet-7in-hdpi-1024-dalvik-heap.mk
index 7fd34b5..e61c40b 100644
--- a/build/tablet-7in-hdpi-1024-dalvik-heap.mk
+++ b/build/tablet-7in-hdpi-1024-dalvik-heap.mk
@@ -18,7 +18,7 @@
 
 PRODUCT_PROPERTY_OVERRIDES += \
     dalvik.vm.heapstartsize=8m \
-    dalvik.vm.heapgrowthlimit=80m \
+    dalvik.vm.heapgrowthlimit=192m \
     dalvik.vm.heapsize=384m \
     dalvik.vm.heaptargetutilization=0.75 \
     dalvik.vm.heapminfree=512k \
-- 
1.8.3.1

